src/js/background/background.js

```diff
@@ -1,8 +1,13 @@
-const RELAY_SITE_ORIGIN = "http://127.0.0.1:8000";
+function startupInit() {
+  const RELAY_SITE_ORIGIN = "http://127.0.0.1:8000";  
+  browser.storage.local.set({ RELAY_SITE_ORIGIN });
+  browser.storage.local.set({ maxNumAliases: 5 });
+  browser.storage.local.set({ relaySiteOrigin: RELAY_SITE_ORIGIN });
```

Comment:
Why set the `RELAY_SITE_ORIGIN` twice w/ different aliases?

Reply:
This is a good call-out. No good reason as to "why" but it's something we can/should fix. 

Filed #537